Source/engine/animationinfo.cpp

```diff
@@ -65,10 +65,13 @@ uint8_t AnimationInfo::getAnimationProgress() const
 	int32_t tickModifier = tickModifier_;
 
 	if (relevantFramesForDistributing_ <= 0) {
+		if (ticksPerFrame <= 0) {
+			Log("getAnimationProgress: Invalid ticksPerFrame {}", ticksPerFrame);
+		}

```

Comment:
```suggestion
		if (ticksPerFrame <= 0) {
			Log("getAnimationProgress: Invalid ticksPerFrame {}", ticksPerFrame);
			return 0;
		}
```

Could we return immediately?
As far as I know this animation is invalid anyway, so the progress can be `0` (0%) or `baseValueFraction` (100%), both right and wrong.
That way we don't have to call `std:max` twice in the good case.

Reply:
I think that'd be a good change, skips doing the rest of the calculations which would be of limited use. Would end up giving an offset of 0,0 when used in the various `CalculateWalkingOffset` functions which is a sensible default result.